app-i18n/transifex-client/transifex-client-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+DESCRIPTION="The Transifex command-line client"
+HOMEPAGE="https://github.com/transifex/cli"
+
+go-module_set_globals
+
+SRC_URI="https://github.com/transifex/cli/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://gitlab.com/api/v4/projects/35204985/packages/generic/${PN}/${PV}/${PN}-1.1-deps.tar.xz"
```

Comment:
Remember you can use bash or `ver_rs` for the `${PN}-1.1-deps.tar.xz` so you don't have to hardcode the version.